src/product-type-import.js

```diff
@@ -30,11 +30,14 @@ const validate = ajv.compile({
 export default class ProductTypeImport {
 
   constructor (logger, { sphereClientConfig, importerConfig }) {
+    const defaultConfig = {
+      rejectOnError: true,
```

Comment:
This is still a breaking change though. This changes the default behaviour 😇

Reply:
Yes :-( ..